src/sentry/uptime/models.py

```diff
@@ -107,6 +108,28 @@ class Meta:
         ]
 
 
+@region_silo_model
+class UptimeSubscriptionRegion(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    uptime_subscription = FlexibleForeignKey("uptime.UptimeSubscription")
+    # The url to check
+    slug = models.CharField(max_length=255)
+    name = models.CharField(max_length=255)
```

Comment:
what are slug and name? how is that related to the url comment?

Reply:
Oops, the comment was a copy/paste leftover.

slug is the slug for the region, name is the more human readable name